Restaurant-Page

teorge opened the pull request:
Completed restaurants-page of UberEats layout

evgenyrodionov commented:
https://erodionov.ru/courses/verstka/FhTKluFzm7

evgenyrodionov commented:
Настрой гитхаб, у тебя коммиты не привязаны к гитхаб-аккаунту

evgenyrodionov commented on restaurant.html:
```diff
@@ -0,0 +1,313 @@
+    <div class="container">
+      <div class="row">
+
+        <div class="col-xs-5 col-md-2 col-lg-3">
```
есть ли необходимость использовать тут сетку?

evgenyrodionov commented on restaurant.html:
```diff
@@ -0,0 +1,313 @@
+      <div class="row">
+
+        <div class="col-xs-5 col-md-2 col-lg-3">
+          <div class="header-logo">
```
а зачем тут этот лишний див?

evgenyrodionov commented on restaurant.html:
```diff
@@ -0,0 +1,313 @@
+        </div>
+
+        <div class="col-md-5 col-lg-5">
+          <div class="location location__desktop">
```
строчкой выше ты оперируешь размером экрана (md, lg), а тут — десктопом. приведи к одному виду

evgenyrodionov commented on restaurant.html:
```diff
@@ -0,0 +1,313 @@
+            <h1 class="restaurant__title">Трактир «Пушкин»</h1>
+            <ul class="tags">
+              <li class="tags__item tags__item_rublesign">
+                &#8381;&#8381;&#8381;
```
да можно так не издрачиваться, у тебя же юникод

evgenyrodionov commented on restaurant.html:
```diff
@@ -0,0 +1,313 @@
+    <nav class="nav">
+      <div class="container">
+        <div class="row">
+          <div class="col-md-12">
```
а смысл тут использовать row / col-md-12? просто вынеси этот список в контейнер и всё

evgenyrodionov commented on restaurant.html:
```diff
@@ -0,0 +1,313 @@
+  <section class="courses">
+    <div class="container">
+      <div class="row">
+        <div class="col-xs-12">
```
то же самое

evgenyrodionov commented on restaurant.html:
```diff
@@ -0,0 +1,313 @@
+      <div class="row">
+        <div class="col-xs-12">
+          <h2 class="courses__title">Закуски</h2>
+          <div class="courses__line"></div>
```
есть тег `hr`

evgenyrodionov commented on src/css/restaurant.css:
```diff
@@ -0,0 +1,918 @@
+  min-width: 302px;
+}
+
+/* Flexbox grid */
```
возьми flexboxgrid2

evgenyrodionov commented on src/css/restaurant.css:
```diff
@@ -0,0 +1,918 @@
+
+.header-logo__img {
+  width: 107px;
+  height: 40px;
```
можно указать только одну сторону (лучше высоту)

evgenyrodionov commented on src/css/restaurant.css:
```diff
@@ -0,0 +1,918 @@
+}
+
+.location__wrapper {
+  width: 227px;
```
только картинкам ставится ширина или высота

teorge commented:
What was done:
- Migration to flexboxgrid2;
- Removed unnecessary grid elements and comments
- Replaced fixed spacing with paddings (box model), also changed unicode symbols from HTML-code to their graphic representation 
- Fixed horizontal rules to hr tag and changed positioning in some elements
- Added package.json and gulpfile.js

evgenyrodionov commented:
теперь осталось определиться как мерджить работу
